frontend/lib/account-settings/route-info.ts

```diff
@@ -4,7 +4,8 @@ export function createAccountSettingsRouteInfo(prefix: string) {
   return {
     [ROUTE_PREFIX]: prefix,
     home: prefix,
-    name: `${prefix}/name`,
+    name: `${prefix}/name`, // This is name rather than legalname to preserve backwards compatibility.
+    preferredname: `${prefix}/preferredname`,
```

Comment:
Our typical naming convention for URL paths is to hyphenate where we'd normally have spaces, so `preferred-name` is preferable here to `preferredname`, but it's not that big a deal.